README.md

```diff
@@ -97,9 +97,41 @@ If you found any of the above modules useful, we would really appreciate if you
 }
 ```
 
+- [4] A. Rosinol, A. Gupta, M. Abate, J. Shi, L. Carlone. [**Kimera: from SLAM to Spatial Perception with 3D Dynamic Scene Graphs**](https://arxiv.org/abs/2101.06894). [arXiv:2101.06894](https://arxiv.org/abs/2101.06894).
+
+```bibtex
+@InProceedings{Rosinol21arxiv-Kimera,
+  title = {{K}imera: from {SLAM} to Spatial Perception with {3D} Dynamic Scene Graphs},
+  author = {A. Rosinol, A. Violette, M. Abate, N. Hughes, Y. Chang, J. Shi, A. Gupta, L. Carlone},
+  year = {2021},
+  booktitle = {arxiv},
+  pdf = {https://arxiv.org/pdf/2101.06894.pdf}
+}
+```
+
+## Open-Source Datasets
+
+We use a photo-realistic Unity-based simulator to test Kimera.
+The simulator provides:
```

Comment:
the description should belong to the datasets themselves: why listing topics here?

Reply:
I might remove the topics, yes, but I would keep the list of info in the dataset to make it easy for the reader to decide if the dataset is of interest. IMHO that is nicer than just providing the link.

Reply:
agreed!